### DISCUSSION TITLE
blank.detect threshold with interactive value

### DISCUSSION THREAD

**Discussion by YoannQueret** [savonet/liquidsoap#3449] [Q&A]
Hi,

Is it possible to use interactive value with blank.detect threshold ?

It seem that the new value is not used.

```
blank_threshold = interactive.float("blank_threshold", -40.0)
[...]
source = blank.detect(
    max_blank=10.0,
    min_noise=10.0,
    on_noise=handler_noise,
    threshold=blank_threshold(),
    track_sensitive=false,
    start_blank=false,
    handler_blank,
    source
)
```

If i change from -40 (value at startup) to -0.5, with an input audio level at -20dB blank.detect detect nothing.

Something is wrong in the code (liquidsoap 2.1.4) ?

Yoann

**Comment by smimram** [comment]
I am adding support for getter arugments in #3452.
